researchstudio-sat/webofneeds
# Feat remove isbranch rebased

### Trigger Warning! Do not get discouraged by the amount of files that have been changed, most of them are simply test-data for our unittests

This branch removes the won:is from every need generation (may it be within a bot, or owner-webapp)

AFAIK there is only one ToDo left, and that is to Update the SparqlMatcherActor.java to support the generic query (this will be handled in issue #2519 by @quasarchimaere  and @peacememories). The Pull Request can still be merged without this though since most of our UseCases do not rely on the generic Query and thus making it less important.

This PR will be needed for any future need-related work(or bot-framework updates that will be necessary for the paypal-bot updates) and should be merged somewhat swiftly.


How To Test:
Basically everything should work as it did before
Please pay special attention to these Parts:
- DebugBot communication and need creation
- RealEstateNeedGenerator
- HokifyJobBot

Another addition in this PR is that we display the need types again, the type has been removed in favor of types in another pr, and thus made the actual need type(s) invisible in the owner-webapp
